src/components/Table/TableHead/TableHead.jsx

```diff
@@ -147,9 +150,52 @@ const TableHead = ({
   closeMenuText,
   lightweight,
   i18n,
+  hasResize,
 }) => {
   const filterBarActive = activeBar === 'filter';
-
+  const [columnWidth, setColumnWidth] = useState({});
+  const columnRef = ordering.map(() => createRef());
+  const columnVar = {
+    index: 0,
+    element: Node,
+    startX: 0,
+  };
+  const mousemoveCallback = e => {
+    const leftColumn = columnRef[columnVar.index].current.getBoundingClientRect();
+    const rightColumn = columnRef[
+      columnVar.index
+    ].current.nextElementSibling.getBoundingClientRect();
+    const mousePosition = e.clientX + columnVar.startX;
+    if (mousePosition >= 50 && mousePosition <= leftColumn.width + rightColumn.width - 50) {
+      columnVar.element.style.left = `${mousePosition}px`;
```

Comment:
columnVar.element.style.left = `${mousePosition}px`;
Here we have implemented the approach which is suggested by Sean. Now column is rederring on release of mouse only.  
 Also we have implemented column dragging liner (the approach which we have shared for which Sean/Luke agreed )
in this implementation in table head we are setting liner (resize div) position through style directly because if we are using state then it start showing performance issue because of re-renderring.  Please suggest on this.